lib.sh

```diff
@@ -110,9 +112,16 @@ ENVASIVE=/etc/apache2/mods-available/mod-evasive.load
 APACHE2=/etc/apache2/apache2.conf
 # Full text Search
 [ ! -z "$ES_INSTALL" ] && ROREST=$(tr -dc "a-zA-Z0-9@#*=" < /dev/urandom | fold -w "$SHUF" | head -n 1)
-[ ! -z "$ES_INSTALL" ] && ES_VERSION=6.1.1
-[ ! -z "$ES_INSTALL" ] && ES_DEB_VERSION="$(echo $ES_VERSION | head -c 1)"
+[ ! -z "$ES_INSTALL" ] && RORESTVERSION="$(curl -s https://api.github.com/repos/sscarduzio/elasticsearch-readonlyrest-plugin/tags | grep "name" | cut -d\" -f4 | head -1)"
+[ ! -z "$ES_INSTALL" ] && ZIPBALL="https://api.github.com/repos/sscarduzio/elasticsearch-readonlyrest-plugin/zipball/$RORESTVERSION"
+[ ! -z "$ES_INSTALL" ] && ES_VERSION="$(echo "$RORESTVERSION" | awk -F'es' '{print $2}')"
+[ ! -z "$ES_INSTALL" ] && ES_DEB_VERSION="$(echo "$ES_VERSION" | head -c 1)"
 [ ! -z "$ES_INSTALL" ] && NCADMIN=$(sudo -u www-data php $NCPATH/occ user:list | awk '{print $3}')
+#Prepare docker env
+[ ! -z "$ES_INSTALL" ] && DOCKER_INS=$(dpkg -l | grep ^ii | awk '{print $2}' | grep docker)
+[ ! -z "$ES_INSTALL" ] && nc_rores6x="ark74/nc_rores6.x:1.6.23_es6.3.2"
+[ ! -z "$ES_INSTALL" ] && rores6x_name="es6.3.2-rores_1.6.23"
+
```

Comment:
Can you remove all the $vars that are not used by the script? I tried but I didn't get the latest version of all the changes...